### DISCUSSION TITLE
setup

### DISCUSSION THREAD

**Discussion by JakeHolcroft** [pluginpal/strapi-plugin-publisher#64] [General]
Once i install the plugin, 
add this to my plugins.js:

module.exports = ({ env }) => ({
  // ..
	'publisher': {
		enabled: true,
		config: {
			hooks: {
				beforePublish: async ({ strapi, uid, entity }) => {
					console.log('beforePublish');
				},
				afterPublish: async ({ strapi, uid, entity }) => {
					console.log('afterPublish');
				},
				beforeUnpublish: async ({ strapi, uid, entity }) => {
					console.log('beforeUnpublish');
				},
				afterUnpublish: async ({ strapi, uid, entity }) => {
					console.log('afterUnpublish');
				},
			},
		},
	},
	// ..
});

should the scheduler be enabled?

thanks

**Comment by ComfortablyCoding** [comment]
The only required part is the enabled section.

You might need to try a clean rebuild for it to show. Instructions on how to do so are in the [note at the end of the usage instructions](https://github.com/ComfortablyCoding/strapi-plugin-publisher#usage)

**Reply by AnkitRastogi238** [reply]
Hi,
I am also unable to see the scheduler in the information sidebar.
Could you please explain the setup more deeply so I can implement the plugin.
Thanks

**Reply by ComfortablyCoding** [reply]
Responded in https://github.com/ComfortablyCoding/strapi-plugin-publisher/discussions/106#discussioncomment-6973128
